updatecli/updatecli.d/vsphere-csi.yml

```diff
@@ -0,0 +1,59 @@
+# This small test makes sure that updatecli is working properly on a repo.
+# To test this:
+#   have "UPDATECLI_GITHUB_ACTOR" env set to your github username
+#   have "UPDATECLI_GITHUB_TOKEN" env set to your github token
+#   have the latest version of updatecli installed
+#   'updatecli diff -v updatecli/values.yaml -c updatecli/updatecli.d/vsphere-csi.yml'
+# In the future, more useful files should be added to this directory.
+---
+name: "Update vsphere csi/cpi charts and images"
+# Make sure we can pull in github repos from multiple orgs
+scms:
+  rke2:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ requiredEnv .github.username }}"
+      token: '{{ requiredEnv .github.token }}'
```

Comment:
Does this work for you? I had problems when using the token from values.yaml

Reply:
it works locally with updatecli and I used [act](https://github.com/nektos/act) as well to test the workflow, it seems to be working fine